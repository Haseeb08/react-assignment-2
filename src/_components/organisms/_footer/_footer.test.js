import React from "react";
import { create } from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";
import Footer from "./_footer";

describe("Footer component testing", () => {
  test("Matches the snapshot", () => {
    const input = create(<Footer />);
    expect(input.toJSON()).toMatchSnapshot();
  });
});
