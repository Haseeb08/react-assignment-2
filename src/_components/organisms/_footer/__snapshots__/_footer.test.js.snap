// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component testing Matches the snapshot 1`] = `
<div
  className="makeStyles-footer-1"
>
  <div
    className="makeStyles-content-2"
    data-testid="footer"
  >
    <div>
      <p
        className="MuiTypography-root makeStyles-text-3 MuiTypography-body1"
      >
        2020 Copyright 
          :  
        <a
          className="MuiTypography-root MuiLink-root MuiLink-underlineHover makeStyles-link-4 MuiTypography-colorPrimary"
          href="https://github.com/Haseeb08/"
          onBlur={[Function]}
          onFocus={[Function]}
        >
          git-haseeb
        </a>
      </p>
    </div>
  </div>
</div>
`;
